specification/compute/resource-manager/Microsoft.Compute/stable/2022-07-02/DiskRP/diskRPCommon.json

```diff
@@ -449,6 +449,10 @@
     "SupportedCapabilities": {
       "type": "object",
       "properties": {
+        "diskControllerTypes": {
+          "type": "string",
+          "description": "DiskControllerTypes will indicate which protocol disks are read by. Options are SCSI | NVMe, SCSI | null"
```

Comment:
we need not display null as option..  Options are 'SCSI' or 'NVMe, SCSI'

Reply:
The disk controllers that an OS disk supports. If set it can be SCSI or SCSI, NVME